llm-finetune/train.py

```diff
@@ -323,6 +344,27 @@ def filter_trainer_args_for_logging(trainer_args: TrainingArguments) -> Dict[str
         "gradient_accumulation_steps": trainer_args.gradient_accumulation_steps,
         "warmup_ratio": trainer_args.warmup_ratio,
     }
+    if other_args.use_lora:
+        if other_args.use_qlora:
+            qlora_args = {
+                "use_qlora": other_args.use_qlora,
+                "bnb_4bit_quant_type": other_args.bnb_4bit_quant_type,
+                "use_double_quant": other_args.use_double_quant,
+                "qlora_bit_length": other_args.qlora_bit_length,
+            }
+            argumets.update(qlora_args)
+        
+        lora_args = {
+        "use_lora": other_args.use_lora,
+        "lora_r": other_args.lora_r,
+        "lora_alpha": other_args.lora_alpha,
+        "lora_target_modules": other_args.lora_target_modules,
+        "lora_dropout": other_args.lora_dropout,
+        "lora_bias": other_args.lora_bias,
```

Comment:
```suggestion
        "use_lora": other_args.use_lora,
        "lora_r": other_args.lora_r,
        "lora_alpha": other_args.lora_alpha,
        "lora_target_modules": other_args.lora_target_modules,
        "lora_dropout": other_args.lora_dropout,
        "lora_bias": other_args.lora_bias,
```
Minor, can you please install recommit hooks and format these files